index.html

```diff
@@ -129,19 +134,22 @@ <h2>Multi-Post<br>Stories</h2>
          </li>
       </ul>   
  </section>
-    <section class="h">
-      <h1>About<br>Myself</h1>
+  <aside>
+    <section>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 1-Main (1)](https://user-images.githubusercontent.com/69638013/165756278-42777035-e545-4670-875d-fe3016e203bb.png)  | ![Screenshot_92](https://user-images.githubusercontent.com/69638013/165756119-732a5ed0-e9e9-4645-ac01-90e5ef772e2e.png)

- [ ] In this section you have two blocks; The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right
- Please adjust widths for the blocks in this section 👍🏿